sdk/webpubsub/Microsoft.Azure.WebPubSub.AspNetCore/CHANGELOG.md

```diff
@@ -1,14 +1,9 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2021-11-24)
```

Comment:
>(2021-11-24) [](http://example.com/codeflow?start=16&length=12)

This library is still in preview and waiting for another PR. No urgent to release this package for this change.

Reply:
There are a couple of changes in the ASP.NET Core project to light up BinaryData end to end.  Any reason not to release it?